hal/src/argon/network/network.cpp

```diff
@@ -173,6 +169,28 @@ int if_init_platform(void*) {
     return 0;
 }
 
+#if HAL_PLATFORM_IF_INIT_POSTPONE
```

Comment:
Should the call to `if_init_platform_postpone(nullptr);` in `main.cpp` be wrapped in `#if HAL_PLATFORM_IF_INIT_POSTPONE ... #endif` as well?

Reply:
Probably, but there is always a default weak empty function anyway and it was like that before.